zulipterminal/ui_tools/views.py

```diff
@@ -1535,31 +1535,38 @@ def __init__(
                     ("Date & Time", date_and_time),
                     ("Sender", msg["sender_full_name"]),
                     ("Sender's Email ID", msg["sender_email"]),
-                    (
-                        "View message in browser",
-                        f"Press {view_in_browser_keys} to view message in browser",
-                    ),
-                    (
-                        "Full rendered message",
-                        f"Press {full_rendered_message_keys} to view",
-                    ),
-                    (
-                        "Full raw message",
-                        f"Press {full_raw_message_keys} to view",
-                    ),
                 ],
             ),
         ]
+
+        # actions for message info popup
+        viewing_actions = (
+            "Viewing Actions",
+            [
+                (
+                    "View in browser",
```

Comment:
Since we're adjusting this, what do you think about 'Open in web browser'?

Reply:
yeah, that makes sense, I have made the changes